pkg/printers/flags_test.go

```diff
@@ -0,0 +1,50 @@
+package printers
+
+import (
+	"testing"
+)
+
+func TestIllegalPackageSourceChecker(t *testing.T) {
```

Comment:
@deads2k I'd like to test this through a call to a printer's PrintObj method, however not sure how I'd inject test package paths into illegalPackageSourceChecker used at PrintObj methods

Reply:
Ah, nevermind, just saw your comment below